### ISSUE TITLE
Not show image after insert

### ISSUE THREAD

**Issue by nlshuynguyen** [stulip/react-native-pell-rich-editor#14]
When I insert a image with base64 data, sometimes it didn't show on webview.
I checked post message and it sent data to webview but don't know why it didn't show.
Can you help me check it. 
```
    onPressAddImage = () => {
        const options = {
            quality: 1.0,
            maxWidth: 500,
            maxHeight: 500,
            storageOptions: {
                skipBackup: true,
            },
        };

        ImagePicker.showImagePicker(options, response => {
            console.log('Response = ', response);

            if (response.didCancel) {
                console.log('User cancelled photo picker');
            } else if (response.error) {
                console.log('ImagePicker Error: ', response.error);
            } else if (response.customButton) {
                console.log('User tapped custom button: ', response.customButton);
            } else {
                // let source = { uri: response.uri };

                // You can also display the image using data:
                let source = { uri: 'data:image/jpeg;base64,' + response.data };
                this.richText.insertImage(source.uri);
                this.richText.blurContentEditor();
            }
        });
    }
```
Link video: https://drive.google.com/file/d/1OnyupxgwNzWkgVJ_5G5Qa3cGEaNImEHv/view?usp=sharing

BTW, this is a great library.

**Comment by astulip** [comment]
If the webview doesn't get the focus, the image is not inserted. This is a known issue.

**Comment by user14045** [comment]
not only image. also for inserting link. the typed link in dialog box won't appeared in rich editor if the editor not get the focus

**Comment by wxik** [comment]
fix on `1.0.8`

**Comment by willeder** [comment]
This issue is still occured on IOS. Does anybody knows how to resolve this?
